letsencrypt/cli.py

```diff
@@ -232,11 +248,44 @@ def _treat_as_renewal(config, domains):
         # configuration file.
         question = None
         if ident_names_cert is not None:
+            # TODO: I bet this question is confusing to people who don't know
+            #       how the backend repreentation of certificates work. The
+            #       distinction is renewal updates existing lineage with new
+            #       cert (eventually maybe preserving the privkey), while
+            #       newcert creates a new lineage. And reinstall doesn't cause
+            #       a new issuance at all.
+            if config.renew_by_default:
+                return "renew", ident_names_cert
```

Comment:
We should also default to "renew" if the cert is within its renewal window, I think.